core/sections/clause_8_ogc-process-description.adoc

```diff
@@ -157,6 +157,26 @@ The following JSON Schema fragment illustrates the use of the `format` key to in
 }
 ----
 
+==== Data classes
+
+One common input type that a process might accept is a https://docs.ogc.org/is/17-069r4/17-069r4.html#_collection_[feature collection] indicating that the process will somehow operate over the items of the collection.  This implies that the process will have certain expections about the structure of the collection with regard to which properties the collection contains, their types, etc.  In order to properly handle any arbitrary input collection a process would need to inspect the structure of the collection to ensure that all the expected properties are present with the expected types, etc.  To alleviate the server from having to perform such a tedious, and perhaps computationally expensive, schema validation step this Standard introduces the concept of the _Data Class_ via the `dataClasses` parameter.  
+
+The value of the `dataClasses` parameter is an array of URIs.  Each URI identifies a predefined set of properties or a sub-schema.  Two data resources tagged with the same data class URI value can be assumed to each contain all the properties defined by the data class.  This equivalence allows a server to quickly validate that an input data resource meets the server's expections in terms of the properties available for processing, their types, etc. simply by comparing data class URI values.  If the data class URI of the input data resource matches one of the data class URIs specified in the description of the process input (via the `dataClasses` parameter) then the server can be assured that the process can operate on that data resource.
```

Comment:
> "The process can "assume the required properties will be available",

Agreeing with this change, but:

> media-type, access-mechanism, formats, schema 

I believe that none of these should involve the _process_ itself (as in the hosted process / execution unit), but rather this would all be handled by the OGC API - Processes engine/implementation. Based on the process description and content negotiation, the API implementation should be able to resolve these things or not before even trying to the application package (e.g., without starting the docker image).

Reply:
@fmigneault I'll change the wording to your suggestion "... the server can assume that the required properties will be available ...".  Is that better?  I don't, in this section, want to get into an entire sidebar about content negotiation, access-mechanisms, etc.)  As you say, data class is just another thing the server needs to resolve before it can operate on the data.